src/main/content/operations.html

```diff
@@ -32,9 +32,9 @@
         <div class="col-md-6 justify-content-center">
             <div class="row">
                 <div class="col centeredCol">
-                    <a href="/operations/kabanero-foundation-setup" target="_blank">
+                    <a href="/operations/kabanero-foundation-setup">
                         <img class="kabaneroLogo" src="/img/installation-symbol.png"><br>
-                        Try It - Install Kabanero Foundation
+                        <a class="col-md-5" id="kabanero-foundation-setup-link" href="/operations/kabanero-foundation-setup/">Try It - Install Kabanero Foundation</a>
```

Comment:
you do not need the extra anchor tag here, since its already in an anchor tag, just add the same attributes to the `<a ...` above

Reply:
since the image is also wrapped in the anchor tag and we're stying the button ourselves it causes the image to have the a weird background which looks like the button 
<img width="1274" alt="Screen Shot 2019-07-29 at 2 31 38 PM" src="https://user-images.githubusercontent.com/39460219/62072919-9ecc0300-b20d-11e9-93cb-8fcec0dd05c4.png">